app/config.py

```diff
@@ -0,0 +1,4 @@
+import json
+
+with open("config.json") as file:
```

Comment:
You should write the full path to the JSON file. It's a good practice to specify the full path to the file to avoid confusion and potential errors. You can use the `os` module to get the path of the current script and then construct the full path using `os.path.join()`. Here is an example: 
python
import os

script_dir = os.path.dirname(__file__)
file_path = os.path.join(script_dir, 'config.json')

with open(file_path) as file:
    config = json.load(file)
```
This way, no matter where you run your script from, it will always find the `config.json` file as long as it is in the same directory as your script.